test/gen-server/ApiServerAccess.ts

```diff
@@ -424,8 +423,11 @@ describe('ApiServerAccess', function() {
         { status: 200, data: null }
       );
 
-      // Invite 4 new users to Chimpy's org. The default limit is 50, so this
+      // Invite 4 new users to Chimpy's org. There is no limit by default, so this
```

Comment:
If there is no limit by default, then shouldn't we NOT stub it here so that the test case confirms the expected default behavior?